docs/conditions.md

```diff
@@ -0,0 +1,73 @@
+# CR Conditions and Readiness Probe
+Conditions are..
```

Comment:
They are also backward compatible. State machine changes (e.g. via status) are not.